workitem/field_definition.go

```diff
@@ -93,6 +93,8 @@ type FieldType interface {
 	// is a string field, then we check if the value v has only one element and
 	// return that instead of the whole list.
 	ConvertToModelWithType(other FieldType, v interface{}) (interface{}, error)
+	// ConvertToString converts the given value to a string representation.
+	ConvertToString(value interface{}) ([]string, error)
```

Comment:
`ConvertToString` should be renamed to `ConvertToStringArray`.

Reply:
Arrrg, my bad. Can you please rename to `ConvertToStringSlice`? This subtle yet important in Go.

Reply:
Fixed 532b987.